planner/core/plan_cache_test.go

```diff
@@ -1081,8 +1081,6 @@ func TestPlanCacheWithLimit(t *testing.T) {
 		{"prepare stmt from 'insert into t select * from t order by a desc limit ?, ?'", []int{1, 2}},
 		{"prepare stmt from 'update t set a = 1 limit ?'", []int{1}},
 		{"prepare stmt from '(select * from t order by a limit ?) union (select * from t order by a desc limit ?)'", []int{1, 2}},
-		{"prepare stmt from 'select * from t where a = ? limit ?, ?'", []int{1, 1, 1}},
-		{"prepare stmt from 'select * from t where a in (?, ?) limit ?, ?'", []int{1, 2, 1, 1}},
```

Comment:
Why are they deleted?

Reply:
The planner generates point plans for these 2 queries and point plans cannot hit the plan cache, so these 2 cases cannot pass the test now.